docs/DeveloperGuide.md

```diff
@@ -510,7 +510,132 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
-**Use case: UC11 - View list of commands**
+**Use case: UC11 - Add client**
+
+**MSS**
+
+1.  User requests to add a client.
+2.  DevEnable adds the client to the client list.
```

Comment:
Client list mentioned in this manner seems to suggest internal details of the system - refer to [AB3 developer guide use cases](https://se-education.org/addressbook-level3/DeveloperGuide.html#use-cases) to see how it is circumvented

Reply:
Understood. Tysm!